tock/employees/migrations/0031_userdata_expected_billable_hours.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.2.12 on 2020-05-07 20:04
+
+from django.conf import settings
+import django.core.validators
+from django.db import migrations, models
+
+def backfill_expected_billable_hours(apps, schema_editor):
```

Comment:
This doesn't look like it's called anywhere, was it intended to run as part of this migration?

Reply:
😲 

Yes - good catch! I was briefly toying with combining the two migrations into one and must have mistakenly removed the call to this